UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Error.cshtml

```diff
@@ -1,10 +1,37 @@
 ﻿@{
     ViewData["Title"] = "Maritime Safety Information";
+    Layout = "~/Views/Shared/_MSILayout.cshtml";
+}
+@section Scripts
+{
+<script type="text/javascript" src="@Url.Content("~/js/Custom/RadioNavigationalWarnings.js")"></script>
 }
```

Comment:
I believe there is no need to include RadioNavigationalWarnings.js on Error page. Is there any specific reason behind this?

Reply:
It is used to handle the title if any global exception occurs